test/test_ecs_monitor.py

```diff
@@ -330,22 +330,17 @@ def describe_services_generator(final_running_count):
 
         event_list = [e for e in events]
 
-        assert len(event_list) == 3
+        assert len(event_list) == 7
```

Comment:
Good point, retrofitting fixture was actually easier than alternative implementation.  Did the same for the other test.

👍